### PR TITLE
Upgrade versions to 6.8.9 and 7.7.0

### PR DESCRIPTION
**Pull request by marco-miller**
`make all` => green `"testcluster"` for both commits.

### CODE REVIEW

File: 7.7/hooks/post_push [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION=$(cat Dockerfile | grep '^ENV VERSION' | cut -d" " -f3)
+TAGS=($VERSION 7)
```
Comment (blacktop) [inline 7.7/hooks/post_push:6]: Can you please remove the 7 from the previous “latest” 7 folder?

Comment (marco-miller) [reply 7.7/hooks/post_push:6]: Sure; should be done now.

Comment (blacktop) [reply 7.7/hooks/post_push:6]: 👍

**Comment by dpursehouse** [comment]
@blacktop version 7.7.0 doesn't seem to have been published on dockerhub yet?

**Comment by marco-miller** [comment]
The 7.7.0 image is not on https://hub.docker.com/r/blacktop/elasticsearch/tags. However the x-pack-7.7.0 is there, as well as the 6.8.9 one. Gerrit needs the 7.7.0 image for one of its currently open changes. Thanks.

**Comment by blacktop** [comment]
my apologies, I always forget that dockerHUB is a manual process for new versions.  I wonder if they have an API now for adding new build config? 🤔 

The new tag is up now

**Pull request merged**